pages/pipelines/command_step.md.erb

```diff
@@ -137,6 +137,17 @@ _Optional attributes:_
       <em>Example:</em> <code>"My reason"</code>
     </td>
   </tr>
+  <tr>
+    <td><code>soft_fail</code></td>
+    <td>
+      Should this step cause the build to fail if it exits with a non-zero status.
```

Comment:
Could we use the same format as we've got down in the retry section? I think the extra type info is handy 😄 
e.g. "A boolean value that defines whether or not this job can be retried manually."

Reply:
Sure thing—I’ll make that change 👌🏼

Reply:
Done! I split out the `exit_status` attr, but left the documentation of the actual `soft_fail` attr as it was (seemed more consistent).